plugins/CreativeControl/config.yml

```diff
@@ -0,0 +1,54 @@
+# Configuration for Remco's Creative Control plugin. Written with love.
+# There is a bypass permission which is creativecontrol.admin, only give this to people you deem trustworthy.
+
+#==============================
+#         Messages
+#==============================
+messages:
+    playerEntersCreative: ""
+
+#==============================
+#         Protection
+#==============================
+
+## Toggle whether or not you'd like to block the following actions triggered by players in creative.
+block:
+  # True because nobody wants to be killed by a creative player.
+  pvp: true
+  # Left false because players might request items.
+  dropItems: false
+  # Left false for same reason as above
+  chests: false
+  itemPickup: false
+  # Could be useful in case a creative player tries to spawn creepers rapidly to kill survival players.
+  spawnEggs: true
+
+## Enable / disable what blocks you are allowed to place.
+block-place-blacklist:
+  enabled: true
+  ### Should be the Material name (1.13 support) for blocks. Use this for reference: http://minecraft-ids.grahamedgecombe.com/
+  ### Blocks placing diamond blocks by default.
+  blacklist:
+  - DIAMOND_BLOCK
+  - IRON_BLOCK
+  - GOLD_BLOCK
+  - DIAMOND_ORE
+  - IRON_ORE
+  - GOLD_ORE
+
+## Enable / disable what blocks you are allowed to break.
+block-break-blacklist:
+  enabled: true
+  ### Should be the numeral IDs for blocks. Use this for reference: http://minecraft-ids.grahamedgecombe.com/
```

Comment:
your config also has an issue here, I am sure you mean material name. the list you linked doesn't even support 1.13 names, here is an updated one https://www.digminecraft.com/lists/item_id_list_pc.php

Reply:
You're right. I was porting this from an old repository and just updating it to 1.13 so I really forgot to move all the configuration stuff. Will solve.